packages/suite/src/support/suite/Resize/index.tsx

```diff
@@ -0,0 +1,19 @@
+import { Props } from './Container';
+import { useEffect } from 'react';
+import debounce from 'debounce';
+
+const Resize = (props: Props) => {
+    useEffect(() => {
+        const handleResize = () => {
+            props.updateWindowSize(window.innerWidth, window.innerHeight);
+        };
+        window.addEventListener('resize', debounce(handleResize, 700));
+        return () => {
+            window.removeEventListener('resize', debounce(handleResize, 700));
```

Comment:
this "removeListener" will not work. 
You need to use exact reference for add and remove listener:

```javascript
const handleResize = debounce(() => {
     props.updateWindowSize(window.innerWidth, window.innerHeight);
 }, 700);

window.addEventListener('resize', handleResize);
window.removeEventListener('resize', handleResize);
```